consensus/types/src/beacon_block_body.rs

```diff
@@ -178,57 +178,71 @@ impl<'a, E: EthSpec, Payload: AbstractExecPayload<E>> BeaconBlockBodyRef<'a, E,
         leaves
     }
 
-    /// Produces the proof of inclusion for a `KzgCommitment` in `self.blob_kzg_commitments`
-    /// at `index`.
+    /// Calculate a KZG commitment merkle proof slowly.
```

Comment:
I think slowly is misleading here, the other method just skips some of the computation and this method does the full computation

Reply:
this was my flippant remark, I'll fix it